pkg/securityscan/alert/prometheusrule.go

```diff
@@ -0,0 +1,74 @@
+package alert
+
+import (
+	"bytes"
+	"text/template"
+
+	"github.com/sirupsen/logrus"
+	meta1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	monitoringv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	k8Yaml "k8s.io/apimachinery/pkg/util/yaml"
+
+	cisoperatorapiv1 "github.com/rancher/cis-operator/pkg/apis/cis.cattle.io/v1"
+	"github.com/rancher/wrangler/pkg/name"
+)
+
+func NewPrometheusRule(clusterscan *cisoperatorapiv1.ClusterScan, clusterscanprofile *cisoperatorapiv1.ClusterScanProfile, imageConfig *cisoperatorapiv1.ScanImageConfig) (*monitoringv1.PrometheusRule, error) {
+	configdata := map[string]interface{}{
+		"namespace":       cisoperatorapiv1.ClusterScanNS,
+		"name":            name.SafeConcatName("rancher-monitoring-cis-alerts", clusterscan.Name),
+		"severity":        imageConfig.AlertsSeverity,
+		"scanName":        clusterscan.Name,
+		"scanProfileName": clusterscanprofile.Name,
+		"alertOnFailure":  clusterscan.Spec.ScheduledScanConfig.ScanAlertRule.AlertOnFailure,
+		"alertOnComplete": clusterscan.Spec.ScheduledScanConfig.ScanAlertRule.AlertOnComplete,
+	}
+	scanAlertRule, err := generatePrometheusRule(clusterscan, "alertingRule.template", "./pkg/securityscan/alert/templates/alertingRule.template", configdata)
```

Comment:
IMO the template name and template files should be constants at the top

Reply:
The naming for `alertingRule.template` is inconsistent with other files since they don't use camel case. You're also defining a PrometheusRule CR in the template, which doesn't represent 1 rule but rather one or more groups with one or more rules.

Should it be `prometheusrule.template`?